gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -14,7 +14,17 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
         assert extension.androidHome != null
     }
 
-    private AndroidCommandPluginExtension createExtension() {
+    void testDefaultEvents() {
+        def extension = createExtension()
+        assert extension.getEvents() == 10000
+    }
+
+    void testDefaultSeed() {
+        def extension = createExtension()
+        assert extension.getSeed() == null
```

Comment:
Yeah... I just wanted to make it very explicit that we expect `null` and not `false` or anything else.